.github/workflows/build_and_test_android.yml

```diff
@@ -106,13 +106,14 @@ jobs:
 
   test:
     # These physical devices are not scalable. Only run on postsubmit for now.
-    if: (! inputs.is-pr)
+    # TODO: Remove before submit
+    # if: (! inputs.is-pr)
```

Comment:
Wasn't this supposed to be removed?

Reply:
Ah... Good catch. Sent out the fix https://github.com/openxla/iree/pull/13473